dotcom-rendering/src/components/YoutubeBlockComponent.importable.tsx

```diff
@@ -223,9 +239,8 @@ export const YoutubeBlockComponent = ({
 				eventEmitters={renderingTarget === 'Web' ? [ophanTracking] : []}
 				format={format}
 				origin={process.env.NODE_ENV === 'development' ? '' : origin}
-				shouldStick={stickyVideos}
+				shouldStick={renderingTarget === 'Web' ? stickyVideos : false}
```

Comment:
Disable sticky videos for apps as they visually pass behind ads which are still natively rendered.